### PR TITLE
B 21036 enforce dodid and emplid main

### PR DESCRIPTION
**Pull request by cameroncaci**
> [!IMPORTANT]
> To resolve merge conflicts and diffs with int, this PR has an upstream of https://github.com/transcom/mymove/pull/13689

## Integration PR
https://github.com/transcom/mymove/pull/13649

### CODE REVIEW

**Review by traskowskycaci — APPROVED** [review]
Looks good!

File: src/pages/Office/CustomerOnboarding/CreateCustomerForm.test.jsx [hunk 1]
```diff
@@ -282,6 +284,57 @@ describe('CreateCustomerForm', () => {
...
+
+    await user.selectOptions(getByLabelText('Branch of service'), 'COAST_GUARD');
+
+    await user.type(getByLabelText('First name'), fakePayload.first_name);
+    await user.type(getByLabelText('Last name'), fakePayload.last_name);
+
+    await user.type(getByLabelText('Best contact phone'), fakePayload.telephone);
+    await user.type(getByLabelText('Personal email'), fakePayload.personal_email);
+
+    await userEvent.type(getByTestId('edipi'), fakePayload.edipi);
```
Comment (r-mettler) [inline src/pages/Office/CustomerOnboarding/CreateCustomerForm.test.jsx:309]: If you do the change requested above will need to update the test ids in the test here as well.

Comment (cameroncaci) [reply src/pages/Office/CustomerOnboarding/CreateCustomerForm.test.jsx]: Resolved by making https://github.com/transcom/mymove/pull/13689 an upstream branch

File: src/pages/Office/CustomerOnboarding/CreateCustomerForm.jsx [hunk 2]
```diff
@@ -262,17 +278,17 @@ export const CreateCustomerForm = ({ userPrivileges, setFlashMessage }) => {
                       label="DoD ID number"
                       name="edipi"
                       id="edipi"
-                      labelHint="Optional"
                       maxLength="10"
+                      data-testid="edipi"
```
[the remark this answers is not in the document]
Comment (cameroncaci) [reply src/pages/Office/CustomerOnboarding/CreateCustomerForm.jsx]: Resolved by making https://github.com/transcom/mymove/pull/13689 an upstream branch

**Review by TevinAdams — APPROVED** [review]

**Review by deandreJones — APPROVED** [review]

**Pull request merged**